### PR TITLE
feat: add Format() to timestamp

### LINKED ISSUE #29: Provide a String() or Format() method for Timestamp struct
We need to provide a String() or Format() function for the Timestamp struct so that when printing the timestamp, user won't need to calculate with the `accuracy` themselves.

### PR DESCRIPTION
**Pull request by ghost**
Resolves #29

### CODE REVIEW

File: timestamp.go [hunk 1]
```diff
@@ -43,3 +46,8 @@ func (t *Timestamp) BoundedAfter(u time.Time) bool {
 	timestampLowerLimit := t.Value.Add(-t.Accuracy)
 	return timestampLowerLimit.After(u) || timestampLowerLimit.Equal(u)
 }
+
+// String returns a string of t as a timestamp range calculated with its accuracy.
+func (t *Timestamp) String() string {
+	return fmt.Sprintf("timestamp range: [%v, %v]", t.Value.Add(-t.Accuracy), t.Value.Add(t.Accuracy))
+}
```
Comment (shizhMSFT) [inline timestamp.go]: How about `Format()`?

Comment (ghost) [reply timestamp.go]: Updated.

File: timestamp.go [hunk 2]
```diff
@@ -43,3 +46,11 @@ func (t *Timestamp) BoundedAfter(u time.Time) bool {
 	timestampLowerLimit := t.Value.Add(-t.Accuracy)
 	return timestampLowerLimit.After(u) || timestampLowerLimit.Equal(u)
 }
+
+// Format returns a string of t in format layout.
+// The output is a timestamp range calculated with its accuracy.
+func (t *Timestamp) Format(layout string) string {
+	lowerBound := t.Value.Add(-t.Accuracy)
+	upperBound := t.Value.Add(t.Accuracy)
+	return fmt.Sprintf("[%v, %v]", lowerBound.Format(layout), upperBound.Format(layout))
```
Comment (shizhMSFT) [inline timestamp.go]: nit:
```suggestion
	return fmt.Sprintf("[%s, %s]", lowerBound.Format(layout), upperBound.Format(layout))
```

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**